.gitignore

```diff
@@ -0,0 +1,101 @@
+node_modules/
+.node_modules/
+built/*
+tests/cases/rwc/*
+tests/cases/test262/*
+tests/cases/perf/*
+!tests/cases/webharness/compilerToString.js
+test-args.txt
+~*.docx
```

Comment:
o_o a docx?

Reply:
jajajjaja....docx? well you can include them 😂

Reply:
Lol I got it from the official Typescript .gitignore, that's why haha. [Link to source](https://github.com/Microsoft/TypeScript/blob/master/.gitignore)